### PR TITLE
Redesign Theme for Apollo Docs

### PR DESCRIPTION
**Pull request by unicodeveloper**

### CODE REVIEW

**Comment by stubailo** [comment]
Good change :shipit:

**Comment by unicodeveloper** [comment]
Lmao @stubailo

**Comment by unicodeveloper** [comment]
First draft @abernix @stubailo @evans

**Comment by stubailo** [comment]
image

A few comments:

1. The highlight color in the sidebar should probably extend all the way across, and not have the arrow anymore
2. The title in the top left looks oddly tall - could be a firefox issue though

Looks a lot better already, just a few more tweaks and I'd say it's a big win!

**Comment by abernix** [comment]
I'm guessing the oddly-tall factor (not just a Firefox thing) is in order to match up the position of the text up with the text on the navigation for apollographql.com itself?  Agree about the other tweaks @stubailo suggested though!

**Comment by unicodeveloper** [comment]
Will work on 1 today @stubailo. For 2. I don't think we need to change anything about that. I agree with @abernix.

**Comment by abernix** [comment]
fyi: For testing purposes, I published this as `meteor-theme-hexo@1.0.11-beta.0` on the `beta` npm dist-tag.

**Comment by abernix** [comment]
Heads up, it appears the search results are misbehaving, as seen on [this Netlify preview](https://deploy-preview-155--apollo-docs-index.netlify.com/docs/): image.

**Comment by unicodeveloper** [comment]
That's interesting. Need to change the style for `algolia-docsearch-suggestion`

File: source/style/_global/docsearch.less [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @color-highlight-header-background: @color-mediumlight;
-@color-highlight-text: @color-primary;
+@color-highlight-text: @color-text-highlight;
```
Comment (abernix) [inline source/style/_global/docsearch.less]: Is it possible that these changes are going to break the Meteor search?  (They're in `_global`.)

File: source/style/_theme/colors.less [hunk 2]
```diff
@@ -168,7 +166,8 @@
       padding: 0;
       margin-top: 0;
       .menu-content {
-        background-color: transparent;
+        //background-color: transparent;
```
Comment (abernix) [inline source/style/_theme/colors.less]: Strong preference to remove commented code unless there's a compelling reason to keep it around.

File: source/style/_theme/colors.less [hunk 3]
```diff
@@ -87,27 +88,24 @@
...
     }
 
     select {
       color: @color-lightest;
     }
 
     .topcap {
-      background-color: darken(#2c2f39, 5%);
+      background: #353c58;
+      //background-color: darken(#2c2f39, 5%);
```
Comment (abernix) [inline source/style/_theme/colors.less]: We might as well remove the commented code here.

File: source/style/_theme/sidebar.less [hunk 4]
```diff
@@ -48,7 +48,7 @@
 		padding-left: 0;
 		margin: 0;
 
-		.std-xpadding;
+		//.std-xpadding;
```
Comment (abernix) [inline source/style/_theme/sidebar.less]: Let's remove it!

**Comment by abernix** [comment]
@unicodeveloper Can we make this not pink for Meteor? image

File: source/style/_global/docsearch.less (hunk 1 above)
Comment (abernix) [reply source/style/_global/docsearch.less]: Nevermind.  Looks okay! 

image

**Comment by abernix** [comment]
This is an incredible update to the Apollo theme.  Thanks so much for implementing this, @unicodeveloper.  And thank you for the beautiful design, @itsjessmarina!

**Pull request merged**